pkg/ddevapp/config.go

```diff
@@ -711,17 +711,22 @@ func (app *DdevApp) CheckCustomConfig() {
 		}
 	}
 
-	customDockerPath := filepath.Join(ddevDir, "web-build")
-	if _, err := os.Stat(customDockerPath); err == nil {
-		dockerFiles, err := filepath.Glob(filepath.Join(customDockerPath, "*Dockerfile*"))
-		util.CheckErr(err)
-		dockerFiles = slices.DeleteFunc(dockerFiles, func(s string) bool {
-			return strings.HasSuffix(s, ".example")
-		})
-		if len(dockerFiles) > 0 {
-			printableFiles, _ := util.ArrayToReadableOutput(dockerFiles)
-			util.Warning("Using custom web-entrypoint.d configuration: %v", printableFiles)
-			customConfig = true
+	for _, buildType := range []string{"web-build", "db-build"} {
+		customDockerPath := filepath.Join(ddevDir, buildType)
+		if _, err := os.Stat(customDockerPath); err == nil {
+			dockerFiles, err := filepath.Glob(filepath.Join(customDockerPath, "Dockerfile*"))
+			util.CheckErr(err)
```

Comment:
I don't really like this usage any more, it seems to hide behavior. Even though the go usage `if err != nil` is ugly and verbose, it's very clear what it does. Would prefer to use that.

Reply:
Yes, it's not obvious, but it's used everywhere in this function, so I didn't change it.

Reply:
OK